### ISSUE TITLE
shell

### ISSUE THREAD

**Issue by Nag-Raju** [koalaman/shellcheck#2688]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/your/shebang
Your script
```

#### Here's what shellcheck currently says:
```
TODO: Describe current output
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by Nag-Raju** [comment]
for subnet in ${subnet_cidr["$@"]}; do
    aws ec2 create-subnet --vpc-id --cidr-block 10.200..0.0/16 --tag-specifications ResourceType=subnet,Tags=[{Key=Name,Value=automation-subnet}] | tr -d '"' | tee SUBNET-Deatails
    SUBID=$(aws ec2 describe-subnets --filters "Name=vpc-Id,Values=$VPCID" | jq ".subnets.subnetId" | tr -d '"')
    echo "**************************************************************************"
    echo "$SUBID"
done

**Comment by Nag-Raju** [comment]
for subnet in ${subnet_cidr["@""]}; do
    aws ec2 create-subnet --vpc-id --cidr-block 10.200..0.0/16 --tag-specifications ResourceType=subnet,Tags=[{Key=Name,Value=automation-subnet}] | tr -d '"' | tee SUBNET-Deatails
    SUBID=$(aws ec2 describe-subnets --filters "Name=vpc-Id,Values=$VPCID" | jq ".subnets.subnetId" | tr -d '"')
    echo "**************************************************************************"
    echo "$SUBID"
done

**Comment by brother** [comment]
Plesae try again. Put code in code blocks as a start.
